src/solver.h

```diff
@@ -155,10 +155,10 @@ struct s_Solver {
    * Solver configuration
    *-------------------------------------------------------------------------------------------------------------*/
 
-  int allowdowngrade;			/* allow to downgrade installed solvable */
-  int allownamechange;			/* allow to change name of installed solvables */
-  int allowarchchange;			/* allow to change architecture of installed solvables */
-  int allowvendorchange;		/* allow to change vendor of installed solvables */
+  int allowdowngrade;			/* allow one to downgrade installed solvable */
+  int allownamechange;			/* allow one to change name of installed solvables */
+  int allowarchchange;			/* allow one to change architecture of installed solvables */
+  int allowvendorchange;		/* allow one to change vendor of installed solvables */
```

Comment:
instead of adding `one`, drop `to` and change the verb (ie. `downgrade` -> `downgrading`, `change` -> `changing`).

Reply:
done